kitsune/sumo/management/commands/merge.py

```diff
@@ -0,0 +1,57 @@
+from kitsune.settings import SUMO_LANGUAGES
+from django.core.management.base import BaseCommand
+from babel.messages.frontend import CommandLineInterface
+
+
+class Command(BaseCommand):
+    """
+    Merge will merge the .pot files into the existing l10n files,
+    which exist in locale (if you have the repo downloaded).
+
+    """
+
+    help = "Merges the .pot files into the existing l10n files."
+
+    def handle(self, *args, **options):
+        no_repo_locales = []
+        for locale in SUMO_LANGUAGES:
+            try:
+                CommandLineInterface().run(
+                    [
+                        "pybabel",
+                        "update",
+                        "-d",
+                        "locale/",
+                        "-l",
+                        locale.replace("-", "_"),
+                        "-D",
+                        "django",
+                        "-i",
+                        "locale/templates/LC_MESSAGES/django.pot",
+                    ]
+                )
+                CommandLineInterface().run(
+                    [
+                        "pybabel",
+                        "update",
+                        "-d",
+                        "locale/",
+                        "-l",
+                        locale.replace("-", "_"),
+                        "-D",
+                        "djangojs",
+                        "-i",
+                        "locale/templates/LC_MESSAGES/djangojs.pot",
+                    ]
+                )
+                self.stdout.write("Merge complete for {}.\n".format(locale))
+            except Exception as e:
```

Comment:
If a locale doesn't have an already existing folder, I think we should use `pybabel init ...` instead of reporting an error. I'm guessing that we'd have to create the `<locale>/LC_MESSAGES/` folder for the locale before calling `pybabel init`, but I'm not sure about that.

Reply:
Would it make sense to punt the init work to an enhancement ticket vs. adding it here? I'm guessing it would be pretty quick effort, but seems more like an addition.  Either way is fine with me, just curious about your thoughts.

Reply:
I think `puente` would do that for locales without existing `.po` files -- [it seems to call `msginit`](https://github.com/mozilla/puente/blob/d2e24cb1c374648313644b7a6d3263ca53b02e5a/puente/management/commands/merge.py#L14) -- so I think it belongs here.

Reply:
Cool. Thanks!